internal/pkg/cli/pipeline_init_test.go

```diff
@@ -612,9 +602,6 @@ func TestInitPipelineOpts_Execute(t *testing.T) {
 				require.EqualError(t, err, tc.expectedError.Error())
 			} else {
 				require.Nil(t, err)
-				require.Equal(t, tc.expectedSecretName, opts.secretName)
-				require.Equal(t, tc.expectManifestPath, opts.manifestPath)
-				require.Equal(t, tc.expectedBuildspecPath, opts.buildspecPath)
```

Comment:
Oh, how come we don't want to assert these?

Reply:
I wrote these tests when I was young. One reason is because the function to be tested (Execute) only returns an error. We usually examine if the return values are expected. This actually makes me realize now we don't need `o. buildspecPath ` and `o. manifestPath`. Good catch!

Reply:
> I wrote these tests when I was young. 

 🤣

Awesome, thanks!